src/client/src/apps/SimpleLauncher/index.tsx

```diff
@@ -18,35 +20,31 @@ const autobahn = new AutobahnConnectionProxy(
 )
 
 type Dependencies = {
-  platform: Platform,
-  pricingService: PricingService,
-  tradeUpdatesStream: Observable<TradesUpdate>,
-  serviceStub: ServiceStubWithLoadBalancer,
-  fdc3: SpotlightFdc3
+  platform: Platform
+  pricingService: PricingService
+  tradeUpdatesStream: Observable<TradesUpdate>
+  serviceStub: ServiceStubWithLoadBalancer
 }
 
 export const SimpleLauncher: React.FC = () => {
-
   const [dependencies, setDependencies] = useState<Dependencies>()
 
   useEffect(() => {
-    (async () => {
+    ;(async () => {
```

Comment:
Is there a reason for the initial semi-colon?

Reply:
This will be prettier doing its thing as RT don't use semi colons - this addition here no doubt prevents automatic semi-colon insertion.